src/Stripe.net/Services/FileLinks/StripeFileLinkCreateOptions.cs

```diff
@@ -0,0 +1,17 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class StripeFileLinkCreateOptions : StripeFileLinkSharedOptions
+    {
+        /// <summary>
+        /// The ID of the file.
+        /// </summary>
+        [JsonProperty("file")]
+        public string File { get; set; }
```

Comment:
The convention in stripe-dotnet is to name things like this with `Id` at the end.

Reply:
Nice catch. Fixed.